cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebViewHelper.java

```diff
@@ -95,6 +99,60 @@ public void run() {
         });
     }
 
+    @TargetApi(11)
```

Comment:
@Undume which API is not exist in level 10? I think you should use java reflection here, or it can not compile if compiling with API 10+.